devicetypes/smartthings/dimmer-switch.src/dimmer-switch.groovy

```diff
@@ -46,7 +46,7 @@ metadata {
 	}
 
 	preferences {
-		input "ledIndicator", "enum", title: "LED Indicator", description: "Turn LED indicator... ", required: false, options:["on": "When On", "off": "When Off", "never": "Never"], defaultValue: "off"
+		input "ledIndicator", "enum", title: "LED Indicator", description: "Turn LED indicator on... ", required: false, options:["on": "When On", "off": "When Off", "never": "Never"], defaultValue: "off"
```

Comment:
I don't think we want the extra ` on` do we?

Reply:
I may have misunderstood, but I think it previously would have read as "Turn LED indicator...[when on]"

Whereas with this fix it should be "Turn LED indicator on...[when on]" which I think more accurately explains what it does

Reply:
I think you're right. Not sure what I was thinking originally